proxy/proxy.go

```diff
@@ -482,22 +488,28 @@ func WithParams(p Params) *Proxy {
 		defaultHTTPStatus = p.DefaultHTTPStatus
 	}
 
+	openTracingInitialSpan := p.OpenTracingInitialSpan
+	if openTracingInitialSpan == "" {
```

Comment:
This can not happen as we have a default from CLI set, or is this helping the skipper use as library?

Reply:
yes, it's for the library use case, potentially for testing, too. Not particularly useful, but it's minimal effort and code.